konrad/humidity.py

```diff
@@ -78,17 +78,21 @@ def get_relative_humidity_profile(self, p):
 
         return rh
 
-    def get_vmr_profile(self, p, T, z):
+    def get_vmr_profile(self, atmos):
```

Comment:
I prefer full variable names as long as their are not too long. But again, this is just a matter of taste